pnpm/src/cmd/complete.ts

```diff
@@ -20,7 +20,7 @@ export async function complete (
     lastOption: string | null
     options: Record<string, unknown>
   }
-) {
+): Promise<Completion[]> {
```

Comment:
@zkochan `@pnpm/tabtab` provides its own type definition of `CompletionItem`, do you want to replace the `Completion` type in pnpm codebase with `CompletionItem`?

Reply:
ok